BringgSDK.js

```diff
@@ -1390,9 +1407,19 @@ var BringgSDK = (function () {
     head.appendChild(script);
   }
 
-  return module;
-}());
-
-
+  function _updateAssetPath(path) {
+    if(!path){
+      return path;
+    }
 
+    var result;
+    if (path.indexOf('http') !== -1) {
+      result = path;
+    }else{
+      result = 'http://app.bringg.com' + path;
```

Comment:
- maybe use https?
- also pretty minot so you can ignore but you can do this if/else and return in one liner -> 
return path.indexOf('http') !== -1 ? path : 'http(s)://app.bringg.com' + path